CodeReviewChallenge.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+
+public class Class1 {
+    public final String SPECIAL_CHARACTER = "a";
+
+    public List<String> method(String paragraph) {
```

Comment:
Change name of method "method" to something more specific e.x "findAllWordsContainingSpecialCharacter"